client/components/SuperHost.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const SuperHost = () => (
+    <div className="textComponent">
+       <img className="topicIcons" src="../img/host.JPG"></img>
+       <div className="wordish">
+       <h3>Dane is a Superhost</h3>
```

Comment:
@pauly-van is this supposed to be hardcoded? or to have data put in later for the name heading?

Reply:
I'll be grabbing an endpoint from Dane i just did this for fun